hs_core/management/commands/solr_test_new_discover.py

```diff
@@ -0,0 +1,109 @@
+"""
```

Comment:
From Alva: this can be deleted. We will develop another one as we revise SOLR configuration.